spec/integrity/i18n_spec.rb

```diff
@@ -25,11 +25,11 @@
     Dir["#{Rails.root}/config/locales/client.*.yml"].each do |f|
       locale = /.*\.([^.]{2,})\.yml$/.match(f)[1]
       client = YAML.load_file("#{Rails.root}/config/locales/client.#{locale}.yml")
-      client.count.should == 1
-      client[locale].should_not == nil
-      client[locale].count.should == 2
-      client[locale]["js"].should_not == nil
-      client[locale]["admin_js"].should_not == nil
+      expect(client.count).to eq(1)
+      expect(client[locale]).not_to eq(nil)
+      expect(client[locale].count).to eq(2)
+      expect(client[locale]["js"]).not_to eq(nil)
```

Comment:
What about changing ``not_to eq(nil)` to `.to be_present`? I think it's better readable.